.changeset/fresh-kings-slide.md

```diff
@@ -0,0 +1,47 @@
+---
+"@spectrum-css/tabs": major
+---
+
+Tab has now been migrated to Spectrum 2. This migration brings with it several major changes:
+
+- Emphasized variant has been removed
+- The divider line has been removed and all variants now resemble what was formerly the quiet variant
+- Removal of t-shirt sizes (only one size is available)
+- Updated high contrast styles
+- The focus indicator's size has changed to accommodate the selection indicator inside of the focus outline
+
+Overflow and vertical variants, aside from what applies from the notes above, remain mostly unchanged and are not fully supported in Spectrum 2. One exception: a bug fix was made to allow density variants to be visible in the overflow variant (previously, the overflow variant was identical for both densities).
```

Comment:
You may already know the answer to this, but _will_ vertical and/or overflow tabs be supported in S2 eventually? Should we hide the stories from the docs page for the time being?

Reply:
Hidden!